docs/source/api-reference/drivers/flashers.md

```
@@ -39,29 +40,31 @@ export:
       port: "1"
 ```
 
-flasher drivers require four children drivers:
-| Child Driver | Description | Auto-created |
-|--------------|-------------|--------------|
-| serial | To communicate with the DUT via serial and drive the bootloader and busybox shell | No |
-| power | To power on and off the DUT | No |
-| tftp | To serve binaries via TFTP | Yes |
-| http | To serve the images via HTTP | Yes |
+flasher drivers require four children drivers: | Child Driver | Description |
```

markdownlint-cli2 (0.17.2)

Heading style
Expected: atx; Actual: setext

(MD003, heading-style)